src/main/cli/settings.c

```diff
@@ -1131,7 +1131,11 @@ const clivalue_t valueTable[] = {
 
     { "angle_level_strength",       VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].P) },
     { "horizon_level_strength",     VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].I) },
-    { "horizon_transition",         VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].D) },
+    { "angle_level_derivative_gain",VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].D) },
+    { "horizon_transition",         VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, horizonTransition) },
+    { PARAM_NAME_ANGLE_FEEDFORWARD, VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].F) },
+    { PARAM_NAME_LEVEL_GYRO_FF_SCALE, VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 100 }, PG_PID_PROFILE, offsetof(pidProfile_t, levelGyroFeedforwardScale) },
```

Comment:
Likewise, now's also a good time to put underscores between variable names, to match the text in the cli.  

In this case, to globally change the variable name `levelGyroFeedforwardScale ` to `level_gyro_feedforward`.  

This simplifies searching for code that uses this variable or CLI entry and is why the majority of new settings are done like that.

I don't think it's necessary to add the word 'scale' because it is implicit.  Many CLI values 'scale' things :-)